### ISSUE TITLE
Why do FindMarkers DE p-values vary significantly between V4 and V5?

### ISSUE THREAD

**Issue by mauritsunkel** [satijalab/seurat#9443]
Hi, 

In Seurat V5 news, it was stated that the DE pseudocount is changed, now performed at the group level rather than cell level (#8462). (Next to min.pct defaults having gone from 0.1 to 0.01 and logfc.threshold from 0.25 to 0.1)
There is no note about p-value calculation having changed at all. 

In V5, especially very lowly expressed genes in either groups (pct.1 || pct.2 < 0.01) now have substantially higher log2FC, however, they also have substantially lower p-val(-adj). In V4, I would filter out a relevant portion of these lowly expressed genes based on having high p-value. 

I would like to run the new V5 estimations for log2FC, yet keep the V4 calculation for p-values, as expected. 
Could it be explained/noted why p-values are changed in V5 from V4 as well? 

Best,
Maus

**Comment by adyzh** [comment]
Hi @mauritsunkel, Thank you for reaching out! We’re always grateful when folks take the time to help make Seurat better :slightly_smiling_face:! Could you send a minimally reproducible example and I can look into this. I believe there are no p-value changes from v4 to v5.

**Comment by rsatija** [comment]
I don't believe that V4 and V5 p-values change. What is happening is that since log2FC values change, additional genes pass the log2FC threshold, and those genes may or may not pass significance thresholds. But the same gene with the same data should give the same p-value in both v4 and v5. If this is not true please reopen this nad provide an example and we will look into it.
